docs/execution-providers/QNN-ExecutionProvider.md

```diff
@@ -20,26 +20,29 @@ be executed by a supported accelerator backend library.
 * TOC placeholder
 {:toc}
 
-## Install Pre-requisites
+## Install Pre-requisites (Building from Source Only)
 
-Download the Qualcomm AI Engine Direct SDK (QNN SDK) from [https://qpm.qualcomm.com/main/tools/details/qualcomm_ai_engine_direct](https://qpm.qualcomm.com/main/tools/details/qualcomm_ai_engine_direct)
+If you build QNN Execution Provider from source, you should first
+download the Qualcomm AI Engine Direct SDK (QNN SDK) from [https://qpm.qualcomm.com/main/tools/details/qualcomm_ai_engine_direct](https://qpm.qualcomm.com/main/tools/details/qualcomm_ai_engine_direct)
 
 ### QNN Version Requirements
 
-ONNX Runtime QNN Execution Provider has been built and tested with QNN 2.18.x and Qualcomm SC8280, SM8350 SOC's
+ONNX Runtime QNN Execution Provider has been built and tested with QNN 2.22.x and Qualcomm SC8280, SM8350, Snapdragon X SOC's
 
 ## Build
 For build instructions, please see the [BUILD page](../build/eps.md#qnn).
 
 ## Pre-built Packages
-Alternatively, ONNX Runtime with QNN EP can be installed from:
+Note: Starting version 1.18.0 , you do not need to separately download and install QNN SDK. The required QNN dependency libraries are included in the OnnxRuntime packages.
 - [NuGet package](https://www.nuget.org/packages/Microsoft.ML.OnnxRuntime.QNN)
-- Nightly Python package (Windows ARM64):
+- [Python package](https://pypi.org/project/onnxruntime-qnn/)
   - Requirements:
-    - Windows ARM64
+    - Windows ARM64 (for inferencing on local device with Qualcomm NPU)
+    - Windows X64 (for quantizing models. see [Generating a quantized model](./QNN-ExecutionProvider.md#generating-a-quantized-model-x64-only))
     - Python 3.11.x
     - Numpy 1.25.2 or >= 1.26.4
-  - Install: `python -m pip install -i https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple/ ort-nightly-qnn`
+  - Install: `pip install onnxruntime-qnn`
+  - Install nightly package `python -m pip install -i https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple/ort-nightly-qnn`
```

Comment:
do we need to provide the nuget nightly feed also?

Reply:
yeah. can point it out.